operations/enable-scheduler-logging.yml

```diff
@@ -0,0 +1,19 @@
+# asactors/scheduler
+- type: replace
+  path: /instance_groups/name=scheduler/jobs/name=scheduler/properties/autoscaler/scheduler?
+  value:
+    application:
```

Comment:
This wipes out the complete configuration of the scheduler.

`path: /instance_groups/name=scheduler/jobs/name=scheduler/properties/autoscaler/scheduler?/application/props`

should be `replace`d

Reply:
thanks ... knew there was a stupid error somewhere 🤣